languages/common-lisp/docs/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,89 @@
-# How to implement an Common Lisp concept exercise
+# How to implement a Common Lisp concept exercise
+
+This document describes how to implement a concept exercise for the
+Common Lisp track. As this document is generic, the following
+placeholders are used:
+
+- `<SLUG>`: the name of the exercise in kebab-case (e.g. `anonymous-methods`).
+
+Before implementing the exercise, please make sure you have a good
+understanding of what the exercise should be teaching (and what not).
+This information can be found in the exercise's GitHub issue. Having
+done this, please read the [Common Lisp concept exercises
+introduction][concept-exercises].
+
+To implement a concept exercise, the following files must be created:
+
+<pre>
+languages
+└── csharp
+    └── exercises
+        └── concept
```

Comment:
I think that a lonely `csharp` got left behind here. It should probably be changed to `common-lisp` like it is in v2.

Reply:
Good catch. I replaced C# but didn't even see that csharp.